webapp/wct.conf.json

```diff
@@ -2,6 +2,20 @@
   "suites": [
     "components/test/*.html"
   ],
+  "plugins": {
+    "local": {
+      "browserOptions": {
+        "chrome": [
+          "headless",
+          "disable-gpu",
+          "no-sandbox"
```

Comment:
What does having the sandbox enabled cause to happen?

Reply:
Chrome refuses to start because of lack of caps, which is in fact a somewhat common problem: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker we need `--cap-add=SYS_ADMIN` or the almighty `--priviledged` when starting Docker, but we can't control how GitHub Actions starts our Docker image.

Reply:
Ack. Any way to document that somewhere? Worst case, in the commit message so it at least shows up if someone goes digging in history.